docs/devel/controllers.md

```diff
@@ -0,0 +1,101 @@
+# Writing Controllers
+
+A Kubernetes controller is simply an optimization on bash script that does repeated lists against the API server.
```

Comment:
Is there any other doc that describes Kube controllers? If yes, link it here. If no, then this needs more info. Mention watching next to listing.